README.md

```diff
@@ -0,0 +1,104 @@
+# FMS HF Tuning
+
+This repo aims to create basic tuning scripts with support for specific models, assumes the use of Hugging Face `SFTTrainer` and PyTorch FSDP only. Our approach to tuning is:
+1. Models are loaded from Hugging Face `transformers` or the `foundation-model-stack` -- models are either optimized to use `Flash Attention v2` directly or through `SDPA`
+2. Hugging Face `SFTTrainer` for the training loop
+3. `FSDP` as the backend for training
+
+## Installation
+
+```
+pip install -r requirements.txt
+pip install -U datasets
```

Comment:
is there a reason datasets isn't just part of requirements.txt?

Reply:
yes, there was a version issue with caching. Replaced `requirements.txt` with the correct version.